cmd/kubectl-gadget/utils/trace.go

```diff
@@ -229,7 +229,12 @@ RetryLoop:
 		nodeErrors := make(map[string]string)
 		for _, i := range results.Items {
 			if i.Status.State == "Completed" || i.Status.State == "Started" {
-				successNodeCount++
+				if i.Status.Output == "" && i.Spec.OutputMode != "Stream" {
```

Comment:
I think this should be `i.Status.Output == "" && i.Spec.OutputMode == "Status"` to only consider the case relevant for us. If we compare against stream then we'll have to put there also other modes that we can support in the future, like `File`.